packages/portal/src/pages/themes/index.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <b-container
+    data-qa="themes"
+  >
+    <ContentHeader
+      :title="pageMeta.title"
+      :description="$t('themes.description')"
+    />
+    <b-row class="flex-md-row pb-5">
+      <b-col cols="12">
+        <b-card-group
+          class="card-deck-4-cols"
+          deck
+          data-qa="theme section"
+        >
+          <ContentCard
+            v-for="theme in themes"
+            :key="theme.identifier"
+            :title="theme.name"
+            :url="{ name: 'themes-all', params: { pathMatch: theme.identifier } }"
+            :image-url="imageUrl(theme.primaryImageOfPage)"
+            :image-content-type="imageContentType(theme.primaryImageOfPage)"
+            :image-optimisation-options="{ width: 510 }"
+            :image-alt="imageAlt(theme.primaryImageOfPage)"
+            :texts="[theme.description]"
+            :show-subtitle="false"
+          />
+        </b-card-group>
+      </b-col>
+    </b-row>
+  </b-container>
+</template>
+
+<script>
+  import ContentHeader from '../../components/generic/ContentHeader';
+  import ContentCard from '../../components/generic/ContentCard';
+  import pageMetaMixin from '@/mixins/pageMeta';
+
+  export default {
+    name: 'ThemesIndexPage',
+    components: {
+      ContentHeader,
+      ContentCard
+    },
+    mixins: [pageMetaMixin],
+
+    data() {
+      return {
+        themes: []
+      };
+    },
+
+    async fetch() {
+      const variables = {
+        locale: this.$i18n.isoLocale(),
+        preview: this.$route.query.mode === 'preview'
+      };
+
+      try {
+        const response = await this.$contentful.query('themeFoyerPage', variables);
+
+        this.themes = response.data.data.themePageCollection.items;
+      } finally {
+        this.$scrollTo && this.$scrollTo('#header');
```

Comment:
Why is this needed?

Reply:
it's not, was copy pasted from the exhibitions hub page which has pagination. 

Pagination for the theme pages has been omitted though as it's not relevant.